code/API_definitions/geofencing-subscriptions.yaml

```
@@ -737,8 +737,10 @@ components:
               $ref: "#/components/schemas/Point"
             radius:
               type: integer
-              description: Expected accuracy for the subscription event of device location in m, from location (radius).
-              minimum: 2000
+              description: |
+                Expected accuracy for the subscription event of device location in m, from location (radius).
```

```suggestion
                Expected accuracy for the subscription event of device location, in meters from `center`.
```
I think the old one was used when this property was called `accuracy` instead of `radius`